cinemy/cinemy/settings.py

```diff
@@ -46,6 +46,7 @@
     "django_q",
     "newsletter",
     "import_movies",
+    "rest_framework.authtoken",
```

Comment:
it's enough to include `rest_framework` - the app name that also includes authtoken